### PR TITLE
docs(alert): remove classic theme stories 

### PR DESCRIPTION
**Pull request by jamime**
This PR replaces #3032

### Proposed behaviour
Removal of classic theme from storybook.

### Current behaviour
Currently, component has classic theme storybook examples.

### Checklist

<del>- [ ] Screenshots are included in the PR </del>
- [x] Carbon implementation and Design System documentation are congruent
- [x] All themes are supported
- [x] Commits follow our style guide
<del>- [ ] Unit tests added or updated </del>
- [x] Cypress automation tests added or updated
- [x] Storybook added or updated
<del>- [ ] Typescript `d.ts` file added or updated </del>

### Additional context
N/A

### Testing instructions
Open storybook.
Check that Alert component has no Classic Theme storybook examples.

### CODE REVIEW

File: src/components/alert/alert.stories.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ const handleOpen = () => {
   action('open')();
 };
 
-function makeStory(name, themeSelector, disableChromatic = false) {
+function makeStory(name, themeSelector, disableChromatic = true) {
```
Comment (mkrds) [inline src/components/alert/alert.stories.js:31]: I think in this case the `disableChromatic` arg along with
`chromatic: {
      disable: disableChromatic
    }`
can be removed.

Comment (mkrds) [reply src/components/alert/alert.stories.js:31]: Or am I wrong here @DlgSHi ?

Comment (DlgSHi) [reply src/components/alert/alert.stories.js:31]: The code change is ok, we don't need `chromatic` check for this story.

**Comment by DlgSHi** [comment]
`Alert` / `Flash` components works fine after deleting `classic` story for `Alert` component. :white_check_mark:

**Pull request merged**